packages/astro/src/core/errors/errors-data.ts

```diff
@@ -45,6 +45,7 @@ export const AstroErrorData = {
 	 * To redirect on a static website, the [meta refresh attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta) can be used. Certain hosts also provide config-based redirects (ex: [Netlify redirects](https://docs.netlify.com/routing/redirects/)).
 	 */
 	StaticRedirectNotAvailable: {
+		// TODO remove
```

Comment:
Just a heads up to not remove errors, but deprecate them (`@deprecated since version x`)